packages/store/tests/store.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Store', () => {
   }
 
   @State<SubSubStateModel>({
-    name: 'baz',
+    name: 'baz_',
```

Comment:
Could you return the changes in this test and write a new one that corrects your error?

Reply:
Agreed, @JasonAtallah please make a copy of an existing test and test your own scenario in a test specifically for that scenario (leaving the existing tests as they were).
Thanks!

Reply:
@splincode, @markwhitfeld I reverted the changes I made to existing tests and created a new test with a new state. The only change I made to the current tests was adding my new state to the root state.